stubs/golang/build.gradle.kts

```diff
@@ -88,8 +88,15 @@ tasks {
         args("install", "github.com/golang/mock/mockgen")
     }
 
+    val copyGoMod by registering(Copy::class) {
+        into("build/generated/proto/main/github.com/infostellarinc/go-stellarstation")
+        from("go.mod")
+        from("go.sum")
+    }
+
     val generateProto by getting(org.curioswitch.gradle.protobuf.tasks.GenerateProtoTask::class) {
         dependsOn(installProtocGoPlugin, installProtoWrap)
+        finalizedBy(copyGoMod)
```

Comment:
Ah interesting, this looks like a good idea.

Can you also run `go mod vendor` in the directory? Go 1.11 users will be able to use this mod/sum but previous versions will not work well unless we create a vendor directory too.

Reply:
I added the task which run ```go mod vendor```. But ```-mod=vendor``` option will be needed when building and testing. I will make changes to support this flag on curioswitch's gradle-golang-plugin.

Reply:
I think you can use `execCustomizer` to override `GOFLAGS` too